basiclti/basiclti-common/src/java/org/sakaiproject/basiclti/util/SakaiBLTIUtil.java

```diff
@@ -2902,6 +2903,117 @@ public static String stripOffQuery(String urlString)
 		return retval;
 	}
 
+	// Copy an LTI content Item from an old site into a new site
+	// assumes legit access to both the old and new site
+	public static Object copyLTIContent(Long contentKey, String siteId, String oldSiteId)
```

Comment:
Comment for the future:
The use if Object is really defeating the purpose of the type system. I've looked through the LtiService and it's full of Object it would be nice to tighten this up.

Reply:
This pattern is used and depended on throughout the LTI code - it is a way that error messages percolate up -  right up into the UI , log message detail and web service response detail.  And yes - I could define a whole bunch of exceptions and turn all the if statements into try / except blocks.  But that would touch a lot of code.   This new code is using underlying code that already follows my "variant return" pattern.  This is not the place to break the pattern.